lib/client-assets.php

```diff
@@ -168,6 +168,10 @@ function gutenberg_register_scripts_and_styles() {
 		filemtime( gutenberg_dir_path() . 'build/compose/index.js' ),
 		true
 	);
+	wp_add_inline_script(
+		'wp-compose',
+		sprintf( 'window._wpSiteURL = %s;', json_encode( site_url() ) )
```

Comment:
Wonder if this is something where we could just assign onto the component prototype itself, rather than having to have this be part of the transform.

i.e.

```php
sprintf( 'wp.components.CodeEditor.siteURL = %s;', json_encode( site_url() ) )
```